src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -685,6 +687,35 @@ bool layout_optimizer::convolution_b_fs_yx_fsv16_opt(const layout& input_layout,
     return false;
 }
 
+static bool has_rank3_mvn_user(const program_node& node, size_t cur_depth, size_t max_depth, uint64_t reorder_size_threshold = 0) {
+    // MVN with rank size 3 always requires Reorder and Reshape. Due to this pattern, too many Reorder may occur when used with Convolution,
+    // which may performance degradation. It stand out in Stable-Diffusion Unet and Decoder.
+    if (cur_depth > max_depth) return false;
+    if (node.is_type<reorder>()) {
+        for (auto& reorder_user : node.get_users()) {
+            if (reorder_user->is_type<reshape>()) {
+                for (auto& reshape_user : reorder_user->get_users()) {
+                    if (reshape_user->is_type<mvn>()) {
+                        if (reorder_size_threshold == 0) {
+                            return true;
+                        } else {
+                            if (node.get_output_layout().get_linear_size() > reorder_size_threshold) {
```

Comment:
Do we need the code at L699 - L701? It looks like that L702 is enough to check the zero threshold condition.